fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/SurveyExperimentIntegrationTest.kt

```diff
@@ -22,7 +22,12 @@ class SurveyExperimentIntegrationTest {
     private val experimentName = "Viewpoint"
 
     @get:Rule
-    val activityTestRule = HomeActivityTestRule()
+    val activityTestRule = HomeActivityTestRule(
+        isJumpBackInCFREnabled = false,
+        isPWAsPromptEnabled = false,
+        isTCPCFREnabled = false,
+        isDeleteSitePermissionsEnabled = true,
```

Comment:
Not sure if you specifically wanted to enable this, as well as the disabling the others.

Reply:
which line exactly?

Reply:
Okay, I removed some of those. I think the one's I kept are all I need.